lib/ansible/utils/module_docs_fragments/aws.py

```diff
@@ -74,7 +81,7 @@ class ModuleDocFragment(object):
     C(AWS_SECRET_ACCESS_KEY) or C(AWS_SECRET_KEY) or C(EC2_SECRET_KEY),
     C(AWS_SECURITY_TOKEN) or C(EC2_SECURITY_TOKEN),
     C(AWS_REGION) or C(EC2_REGION)
-  - Ansible uses the boto configuration file (typically ~/.boto) if no
+  - Ansible uses the boto configuration file (typically ~/.boto or ~/.aws) if no
     credentials are provided. See http://boto.readthedocs.org/en/latest/boto_config_tut.html
```

Comment:
`U(http://boto.readthedocs.io/en/latest/boto_config_tut.html)`
Use `U(..)`, also seems to have moved to `.io`, rather than `.org`

Reply:
Good catch. I also added the boto3 link.

Reply:
I had a PR for the same - #32020. I will go ahead and close it. :)